lib/galaxy/tool_util/deps/__init__.py

```diff
@@ -118,6 +126,7 @@ class DependencyManager:
     dependency available in the current shell environment.
     """
 
+    _destination_for_container_type: Dict[ContainerType, List[DestinationProtocol]]
```

Comment:
No need to annotate it in 2 different places (line 157).

Reply:
I'd prefer to keep this and drop the other declaration. Is that okay? I like to see the declarations and the types outside of the constructor. You shouldn't have to parse the constructor to understand types IMO.

Reply:
Sure, I'd just avoid the duplication.